drivers/flash/Kconfig.cc13xx_cc26xx

```
@@ -11,3 +11,14 @@ config SOC_FLASH_CC13XX_CC26XX
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 	help
 	  Enables TI SimpleLink CC13xx/CC26xx flash controller driver.
+
+config CC13XX_CC26XX_FLASH_TO_FLASH_SUPPORT
+	bool "Enable support of flash to flash write operation"
```

Coding guideline checks are complaining about this
```suggestion
	bool "Flash to flash write"
```